.github/workflows/CI.yml

```diff
@@ -27,16 +27,15 @@ jobs:
           - x64
         suite:
           - '["AlexNet", "VGG"]'
-          - '["GoogLeNet", "SqueezeNet"]'
-          - '["EfficientNet", "MobileNet"]'
-          - '[r"/*/ResNet*", "ResNeXt"]'
-          - 'r"/*/Inception/Inceptionv*"'
-          - '["InceptionResNetv2", "Xception"]'
+          - '["GoogLeNet", "SqueezeNet", "MobileNet"]'
+          - '["EfficientNet"]'
+          - 'r"/^ResNet\z/"'
```

Comment:
Does this not work?
```suggestion
          - 'r"/*/ResNet*"'
```

Reply:
That takes in _all_ the ResNets - including SEResNet and InceptionResNetv2. I want just the vanilla ResNets